cmd/process-agent/config_test.go

```diff
@@ -177,45 +173,42 @@ func TestPodCheck(t *testing.T) {
 
 	t.Run("enabled", func(t *testing.T) {
 		cfg.Set("orchestrator_explorer.enabled", true)
+		cfg.Set("cluster_name", "test")
 
-		ocfg := oconfig.NewDefaultOrchestratorConfig()
-		ocfg.KubeClusterName = "test" // We can't reliably detect a kubernetes cluster in a test
-		assert.NoError(t, ocfg.Load())
+		// ocfg := oconfig.NewDefaultOrchestratorConfig()
+		// ocfg.KubeClusterName = "test" // We can't reliably detect a kubernetes cluster in a test
+		// assert.NoError(t, ocfg.Load())
```

Comment:
Are these lines meant to be commented or deleted?

Reply:
Oops, I will delete these!